tests/helpers/resolver.js

```diff
@@ -1,11 +1,11 @@
-import Resolver from "ember/resolver"
-import config from "../../config/environment"
+import Resolver from '../../resolver';
+import config from '../../config/environment';
 
-var resolver = Resolver.create()
+const resolver = Resolver.create();
 
 resolver.namespace = {
   modulePrefix: config.modulePrefix,
-  podModulePrefix: config.podModulePrefix,
-}
+  podModulePrefix: config.podModulePrefix
```

Comment:
You can remove this line. It was added when I ugpraded and is not something we use.